src/azure-cli/azure/cli/command_modules/acs/decorator.py

```diff
@@ -478,11 +553,16 @@ def _get_dns_name_prefix(
         # In case the user does not specify the parameter and it meets the conditions of automatic completion,
         # necessary information is dynamically completed.
         if dynamic_completion:
-            dns_name_prefix = _get_default_dns_prefix(
-                name=self.get_name(),
-                resource_group_name=self.get_resource_group_name(),
-                subscription_id=self.get_intermediate("subscription_id"),
-            )
+            name = self.get_name()
+            resource_group_name = self.get_resource_group_name()
+            subscription_id = self.get_subscription_id()
+            # Use subscription id to provide uniqueness and prevent DNS name clashes
```

Comment:
original behaviour?

Reply:
Yes, this is the original behavior defined in `_get_default_dns_prefix`.